device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/BALANCED/pg_profile_lookup.ini

```diff
@@ -4,20 +4,20 @@
   25000  5m   1248    2288 53248   0         2288
   40000  5m   1248    2288 66560   0         2288
   50000  5m   1248    2288 90272   0         2288
- 100000  5m   18796   3556 300990  0        3556
- 200000  5m   18796   3556 300990  0        3556
- 400000  5m   18796   3556 300990  0        3556
+ 100000  5m   18796   0    300990  0        3556
+ 200000  5m   18796   0    300990  0        3556
+ 400000  5m   18796   0    300990  0        3556
   10000  40m  1248    2288 37024   0         2288
   25000  40m  1248    2288 53248   0         2288
   40000  40m  1248    2288 71552   0         2288
   50000  40m  1248    2288 96096   0         2288
- 100000  40m  18796   3556 300990  0        3556
- 200000  40m  18796   3556 300990  0        3556
- 400000  40m  18796   3556 300990  0        3556
+ 100000  40m  18796   0    300990  0        3556
+ 200000  40m  18796   0    300990  0        3556
+ 400000  40m  18796   0    300990  0        3556
   10000  300m 1248    2288 46176   0         2288
   25000  300m 1248    2288 79040   0         2288
   40000  300m 1248    2288 108160  0         2288
   50000  300m 1248    2288 141856  0         2288
- 100000  300m 18796   3556 300990  0        3556
- 200000  300m 18796   3556 300990  0        3556
- 400000  300m 18796   3556 300990  0        3556
+ 100000  300m 18796   0    300990  0        3556
+ 200000  300m 18796   0    300990  0        3556
+ 400000  300m 18796   0    300990  0        3556
```

Comment:
@Pterosaur Could you update xon value as '0' for all the speed/cable types to make it consistent for TH5.

Reply:
@Pterosaur Could you check the other TH5 SKUs like 7060x6_64de. We can update all PG ini files with XON value 0.

Reply:
Done. The pg_profile of 7060x6_64de is empty, so, I think, we don't really need to change it.

Reply:
@Pterosaur Valid buffer settings are only for 100m cable length(max) and 200G. Please keep only 5m/40m cable length and 100G/200G speed and remove all the others.